packages/pages/src/bin/spawn.ts

```diff
@@ -13,10 +13,14 @@ const pathToPagesScript = path.resolve(
   "pages.js"
 );
 
-spawnSync(
+const results = spawnSync(
   "node",
   ["--experimental-vm-modules", pathToPagesScript, ...process.argv.slice(2)],
   {
     stdio: "inherit",
   }
 );
+
+if (results?.status != undefined && results.status !== 0) {
```

Comment:
Pretty sure you can just do results?.status !== 0

Reply:
TypeScript doesn't seem to like that
<img width="634" alt="image" src="https://github.com/yext/pages/assets/115099870/907e2980-3f32-4994-8c0d-82be4898f7c0">

Reply:
Hmm ok